src/main/java/com/soen343/gms/model/Task.java

```diff
@@ -0,0 +1,28 @@
+package com.soen343.gms.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Entity
+@Getter
+@Setter
+public class Task
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    long id;
+
+    String taskDescription;
+    boolean isComplete;
+
+    @Override
+    public String toString() {
+        return "Task{" +
+                "id=" + id +
+                ", taskDescription='" + taskDescription + '\'' +
+                ", isComplete=" + isComplete +
+                '}';
+    }
```

Comment:
Nice JSON toString, but as mentioned earlier this most likely will not be needed. Keep it since it shouldn't interfere with application.